libs/typescript-project-references/package.json

```diff
@@ -9,11 +9,12 @@
     "executors": "./executors.json",
     "dependencies": {
         "execa": "^5.1.1",
-        "tsup": "^5.12.9"
+        "tsup": "^8.1.0"
```

Comment:
Does this need to be a dependency, or should it be a peerDependency instead? (non-blocking)

Reply:
Good shout, I've made this change :)